app/models/participant_outcome.rb

```diff
@@ -6,6 +6,8 @@ class ParticipantOutcome < ApplicationRecord
   validates :completion_date, presence: true
   validate :completion_date_not_in_the_future
 
+  delegate :user, :lead_provider, to: :declaration
```

Comment:
If we add `:course` here as well, in the [serializer](https://github.com/DFE-Digital/npq-registration/pull/1490/files#diff-b9d35c56d14224cecdd555788c30d3ca50d92a62852df03a4a65fd1250f09463R11) we could do:

`field(:course_identifier) { |outcome| outcome.course.identifier }`